### PR TITLE
View contact detail

### PR DESCRIPTION
**Pull request by ericmoraess**
View a contact detail on item click.

### CODE REVIEW

File: AgendaUWP/AgendaUWP/ViewModels/ContactListPageViewModel.cs [hunk 1]
```diff
@@ -1,27 +1,63 @@
...
+            Data = GetData();
+            InitilizeCommands();
         }
 
-        private void GetData()
+        private ObservableCollection<ContactItemsGroup> GetData()
         {
             var contacts = _contactService.GetContacts();
-            GroupedContactItems = contacts.GroupBy(m => m.FullName[0] , (key, list) => new ContactItemsGroup(key, list));
+            if (contacts != null && contacts.Count() > 0)
```
Comment (soutolucas) [inline AgendaUWP/AgendaUWP/ViewModels/ContactListPageViewModel.cs:55]: Is it really necessary? If contacts were empty return an empty ObservableCollection, what do you think?

File: AgendaUWP/AgendaUWP/ViewModels/DetailContactPageViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+using Service.Model;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AgendaUWP.ViewModels
+{
+    public class DetailContactPageViewModel : ViewModelBase
```
Comment (soutolucas) [inline AgendaUWP/AgendaUWP/ViewModels/DetailContactPageViewModel.cs:15]: Could you please fix indentation and blank lines?

**Review by soutolucas — CHANGES_REQUESTED** [review]

File: AgendaUWP/AgendaUWP/ViewModels/ContactListPageViewModel.cs (hunk 1 above)
Comment (ericmoraess) [reply AgendaUWP/AgendaUWP/ViewModels/ContactListPageViewModel.cs:55]: Ok! Done!

File: AgendaUWP/AgendaUWP/ViewModels/ContactListPageViewModel.cs [hunk 3]
```diff
@@ -1,27 +1,63 @@
...
+            {
+                GroupedContactItems = contacts.GroupBy(m => m.FullName[0], (key, list) => new ContactItemsGroup(key, list));
+                return new ObservableCollection<ContactItemsGroup>(GroupedContactItems);
```
Comment (lhohenrique) [inline AgendaUWP/AgendaUWP/ViewModels/ContactListPageViewModel.cs]: Please, avoid multiple returns

File: AgendaUWP/AgendaUWP/ViewModels/DetailContactPageViewModel.cs (hunk 2 above)
Comment (ericmoraess) [reply AgendaUWP/AgendaUWP/ViewModels/DetailContactPageViewModel.cs:15]: Ok! Done!

**Review by lhohenrique — CHANGES_REQUESTED** [review]

**Review by soutolucas — APPROVED** [review]

File: AgendaUWP/AgendaUWP/ViewModels/ContactListPageViewModel.cs (hunk 3 above)
Comment (ericmoraess) [reply AgendaUWP/AgendaUWP/ViewModels/ContactListPageViewModel.cs]: Ok! I create a new variable to store the result collection and avoid multiples returns.

**Review by lhohenrique — APPROVED** [review]

**Pull request merged**